boggle_blast.lic

```diff
@@ -0,0 +1,165 @@
+custom_require.call(%w[common common-healing common-items common-money common-travel drinfomon])
+
+class BoggleBlast
+  include DRC
+  include DRCH
+  include DRCI
+  include DRCM
+  include DRCT
+
+  def initialize
+    @stow_container = "backpack" # set this to be your main default storage container - i.e. the one used in STORE LIST set to default
+    @full_container = "haversack" # this container is used to store full 100k HE ticket stacks, set it to a DIFFERENT container than the default
+    @currency = "dokoras" # don't change this
+    @withdraw = true # change to false if you don't want to withdraw more money when you run out
+    @portal_room = 2780 # 2780 = Shard, 932 = Crossing, no clue about the others
```

Comment:
💡 This would be a good candidate for [base-town](https://github.com/rpherbig/dr-scripts/blob/master/data/base-town.yaml) (and maybe `@currency` too)

Reply:
These would be good candidates, but given the temporary nature of this script I wasn't sure if it made sense to add them as such. Your call, I'm fine either way.